frontend/src/pages/employee/EmployeeLogin.js

```diff
@@ -1,7 +1,90 @@
-import React from "react";
-
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import ButtonPrimary from "../../components/UI/ButtonPrimary";
+import InputField from "../../components/UI/InputField";
+import axios from "axios";
+import { toast } from "react-toastify";
 const EmployeeLogin = () => {
-  return <div>EmployeeLogin</div>;
+  const navigate = useNavigate();
+  const [disabled, setDisabled] = useState(false);
+  const [userInfo, setUserInfo] = useState({
+    email: "",
+    password: "",
+  });
+
+  const handleChange = (e) => {
+    setUserInfo({ ...userInfo, [e.target.name]: e.target.value });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setDisabled(true);
+
+    axios
+      .post(`http://localhost:5000/api/employee/login`, {
+        email: userInfo.email,
+        password: userInfo.password,
+      })
+      .then((res) => {
+        if (res.data.success) {
+          // save in localstorage: make sure to save the user type as otherwise can't access protected routes
+          localStorage.setItem("email", res.data.employee.email);
+          localStorage.setItem("token", res.data.token);
+          localStorage.setItem("type", "admin");
+          setDisabled(false);
+          toast.success("Login Successful");
+
+          // change the route to the admin page
+          navigate("/admin/home");
```

Comment:
here you need to change this to go the employee home page /employee/home or maybe /employee/[employee_id] but this one is harder to do so just do /employee/home